care/emr/management/commands/load_fixtures.py

```
@@ -1,1886 +1,335 @@
+                    email=f"{user_def['username']}@care.test",
+                    password=password,
+                )
+                base.assign_org_role(managing_org_id, user["id"], role_id)
+                log(f"  {user_def['role']:<10} {user_def['username']:<25} {password}")
+
+            elif user_def["action"] == "assign":
+                try:
+                    user_data = base.get_user(user_def["username"])
+                    base.assign_org_role(managing_org_id, user_data["id"], role_id)
+                    log(f"  Assigned {user_def['role']} to {user_def['username']}")
+                except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)